src/js/actions/organizationActions.js

```diff
@@ -25,7 +25,7 @@ export const createOrganizationTrial = data => dispatch =>
       }
     })
     .then(res => {
-      cookies.set('JWT', res.text, { maxAge: 900, sameSite: 'strict', path: '/' });
+      cookies.set('JWT', res.text, { maxAge: 900, sameSite: 'strict', path: '/ui' });
```

Comment:
We cannot do this because it breaks deviceconnect and websockets: when we connect to the API end-point which establishes a WebSocket connection, we cannot set additional headers because of limitations in the browser's implementation of WebSockets. We use Cookies to transfer the JWT token, but if we set the Cookie path to /ui it won't be transferred when calling /api/ end-points.